web/src/components/navbar/desktop/DesktopLoginButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { triggerSignIn } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { ReactElement } from "react";
+
+export const DesktopLoginButton = (): ReactElement => {
+  const { Config } = useConfig();
+  return (
+    <span className="margin-right-2">
+      <UnStyledButton
+        dataTestid="login-button"
+        onClick={(): void => {
+          triggerSignIn();
+          analytics.event.landing_page_navbar_log_in.click.go_to_myNJ_login();
```

Comment:
pebble - are we testing the analytics firing or the trigger sign in function being called?

Reply:
Nope, I ripped this out of another function and thought we had tests but I double checked and we do not. So I will add, nice catch